Makefile

```diff
@@ -10,7 +10,9 @@ TAG = lorax-$(VERSION)-$(RELEASE)
 IMAGE_RELEASE = rhel8-latest
 
 ifeq ($(TEST_OS),)
-TEST_OS = rhel-8-1
+OS_ID = $(shell awk -F= '/^ID/ {print $$2}' /etc/os-release)
```

Comment:
The `^ID` expression matches to `ID` and `ID_VERSION` at the same time, so it needs to be more specific to just match only `ID` (`/^ID=/` should be enough).

Reply:
I can see this came from master branch, but this issue doesn't exist there as `/etc/os-release` in Fedora doesn't contain the `ID_LIKE` variable.